### PR TITLE
Allow RA credentials to be both RA(A) and SRAA

### PR DESCRIPTION
**Pull request by DRvanR**

### CODE REVIEW

**Comment by DRvanR** [comment]
:pray: @rjkip

File: src/Surfnet/StepupMiddleware/ApiBundle/Identity/Value/RegistrationAuthorityCredentials.php [hunk 1]
```diff
@@ -123,6 +123,14 @@ private static function assertEquals($nameId, $identityNameId)
     }
 
     /**
+     * Grants the SRAA
+     */
+    public function grantSraa()
+    {
+        $this->isSraa = true;
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Identity/Value/RegistrationAuthorityCredentials.php]: Meh, mutable VO? I'd opt for a new instance.

**Comment by DRvanR** [comment]
:pray: @rjkip

**Comment by rjkip** [comment]
:+1: :+1: :+1:

**Pull request merged**